<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateKeyDialog</class>
 <widget class="QDialog" name="GenerateKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate a key</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Key parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Your name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_name">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>John Doo</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Passphrase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_password">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type it again</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_password_bis">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_errors">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>GenerateKeyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>GenerateKeyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_name</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GenerateKeyDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_password</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GenerateKeyDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_password_bis</sender>
   <signal>textEdited(QString)</signal>
   <receiver>GenerateKeyDialog</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check()</slot>
 </slots>
</ui>
